src/citrine/informatics/predictors/ingredients_to_simple_mixture_predictor.py

```diff
@@ -50,6 +59,11 @@ def __init__(self,
                  id_to_quantity: Mapping[str, RealDescriptor],
                  labels: Mapping[str, Set[str]],
                  archived: bool = False):
+        warn("{this_class} has been renamed. Please use {replacement} instead"
+             .format(this_class=self.__class__.__name__,
+                     replacement="Ingredients To Formulation Predictor"),
```

Comment:
```suggestion
                     replacement=IngredientsToFormulationPredictor.__name__),
```
But you'll need to replace line 7 with
```
from citrine.informatics.predictors import Predictor, IngredientsToFormulationPredictor
```

Reply:
Importing `IngredientsToFormulationPredictor` causes a bunch of tests to fail. I don't know why